sdk/src/types/api/core.rs

```diff
@@ -514,14 +518,6 @@ pub struct UtxoChangesFullResponse {
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct OutputWithId {
```

Comment:
?

Reply:
What are you confused about? I don't see it. Maybe wrong selection?